bundles/org.openhab.binding.touchwand/src/main/java/org/openhab/binding/touchwand/internal/discovery/TouchWandControllerDiscoveryService.java

```diff
@@ -70,7 +71,7 @@ protected void stopScan() {
     }
 
     @Override
-    public void activate(@Nullable Map<String, Object> configProperties) {
+    public void activate(@Nullable Map<@NonNull String, @NonNull Object> configProperties) {
```

Comment:
Your class is already annotated with `@NonNullByDefault` so this change shouldn't be necessary.